code/modules/materials/_materials.dm

```diff
@@ -355,21 +355,29 @@ INITIALIZE_IMMEDIATE(/obj/effect/gas_overlay)
 
 // Generic material product (sheets, bricks, etc). Used ALL THE TIME.
 // May return an instance list, a single instance, or nothing if there is no instance produced.
-/decl/material/proc/create_object(var/atom/target, var/amount = 1, var/object_type, var/reinf_type)
+/decl/material/proc/create_object(var/atom/target, var/amount = 1, var/object_type, var/reinf_type, var/always_return_as_list = FALSE)
```

Comment:
`always_return_as_list` doesn't seem to be used?

Reply:
Beh I had some rebase problems, meant to remove that.